---
input:
    color: 1  # 0 for grayscale, 1 for color
    frame_id: "depth_optical_frame"
path:
    num_of_knots: 25
    num_of_pts: 256
    max_width: 40
    min_points: 3
    min_length: 20
    width_step: 4
    vector_dir_len: 5
    between_grippers: False  # Set to true if you want to receive a spline between horizontally oriented grippers
hsv:
    hue:
        min: 176
        max: 7
    saturation:
        min: 130
        max: 255
    value:
        min: 0
        max: 255
depth:
    min: 200
    max: 650
